test/integration/taxon_browsing_test.rb

```diff
@@ -396,6 +403,13 @@ def and_i_can_see_an_email_signup_link
     )
   end
 
+  def then_i_cannot_see_an_email_signup_link
+    assert page.has_no_link?(
+      'Get email alerts for this topic',
+      href: "/email-signup/?topic=#{current_path}"
+    )
+  end
+
```

Comment:
The email sign up link appears for in each accordion section, see: https://www.gov.uk/education/funding-and-finance-for-students#/education/financial-help-for-students-who-are-parents-or-carers

Should we also check that email sign up links are not rendered for each section of the accordion when the taxon is not live?. The opposite of what the method on line 413 is testing for.

Reply:
Good spot 👍